src/mame/video/namcos22.cpp

```diff
@@ -530,7 +530,8 @@ struct namcos22_scenenode *namcos22_renderer::alloc_scenenode(running_machine &m
 	}
 	else
 	{
-		node = auto_alloc(machine, struct namcos22_scenenode);
+		m_scenenode_alloc.emplace_back();
+		node = &m_scenenode_alloc.back();
```

Comment:
Another case of an unnecessary extra dereference (`emplace_back` returns a reference as of C++17).